pkg/controller/bootstrap/util.go

```diff
@@ -17,36 +17,40 @@ limitations under the License.
 package bootstrap
 
 import (
+	"fmt"
 	"time"
 
-	"k8s.io/klog/v2"
+	"golang.org/x/net/context"
```

Comment:
Maybe  "context"?

Reply:
Oops! the IDE automatically imported the incorrect context, I didn't notice. Thanks!